Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -87,6 +87,35 @@ configuration:
   name: close_ticket
   required: false
   type: 8
+- defaultvalue: None
```

Comment:
lets change a bit the params order:
below fetch params place the mirroring params, where the new ones are the first and then the Close params 
below the mirror params place the insecure and proxy